### DISCUSSION TITLE
Email from LG

### DISCUSSION THREAD

**Discussion by leppy1986** [nVuln/homebridge-lg-thinq#135] [Q&A]
Hi,

I received an email from LG today stating:

> The same unusual request is being made repeatedly from your account.
> If you are using the ThinQ service in a form other than an officially distributed app, please stop immediately.
> If it is repeated over and over again, it may be judged as an abnormal user and the use of the app may be restricted.

It appears to be genuine - I assume it relates to this plugin. Has anyone else received this?

**Comment by nVuln** [comment]
is your device using thinq1 ?

**Reply by leppy1986** [reply]
No
